### PR TITLE
More complete description of zkRollups

### LINKED ISSUE #2797: More complete description of ZK rollups
**Is your feature request related to a problem? Please describe.**

I very like the description of the L2 solutions as it lets the Ethereum community know more about the different ways Ethereum can be scaled today. However, I find the description somewhat biased against zero knowledge rollups :slightly_frowning_face: 

The section that describes ZK rollups says that they are "_Limited to simple transfers, not compatible with the EVM._", which is not true at all.

StarkWare is working on [Cairo & StarkNet](https://medium.com/starkware/on-the-road-to-starknet-a-permissionless-stark-powered-l2-zk-rollup-83be53640880). Even though Cairo is not EVM-compatible, it is a non-Turing-complete language that will allow much more than simple transfers.

Matter Labs is working on [Solidity support & near-full EVM compatibility](https://medium.com/matter-labs/zksync-2-0-roadmap-update-zkevm-testnet-in-may-mainnet-in-august-379c66995021). This Spring, even before the launch of L2 smart contracts, zkSync will natively support NFTs and atomic swaps.

Also, the description of Optimistic Rollups says that "_All transaction data is stored on the layer 1 chain, meaning it's secure and decentralized._", which is absolutely correct. I believe that we should also mention that in ZK rollups "_The data that is needed to restore the state is stored on the layer 1 chain, meaning that it's secure and decentralized_" because that is the point of the word "rollup". The data is stored on-chain to protect the users of the protocol.

**Describe the solution you'd like**

I propose to change the wording:

From "_Limited to simple transfers, not compatible with the EVM._" to "_Some don't have full EVM support._".

Also, add "_The data that is needed to restore the state is stored on the layer 1 chain, meaning that it's secure and decentralized_" as one of the Pros of ZK rollups.

**Describe alternatives you've considered**

I believe that any description of ZK rollups that is both accepted by the community and fairly describes their full potential that goes far beyond "simple transfers" is a good one :slightly_smiling_face:.

**Want to contribute?**

Yes, but I would really love :heart: to hear what the community thinks about the proposal and if I missed something.

### PR DESCRIPTION
**Pull request by StanislavBreadless**
## Description

Added more complete description of zero-knowledge rollups' pros and cons

## Related Issue

Closes #2797

### CODE REVIEW

File: src/content/developers/docs/layer-2-scaling/index.md [hunk 1]
```diff
@@ -69,17 +69,17 @@ The sidechain where ZK rollups happen can be optimised to reduce transaction siz
...
+| Faster finality time since the state is instantly verified once the proofs are sent to the main chain.            | Some don't have EVM support.                                                                          |
+| Not vulnerable to the economic attacks that [Optimistic rollups](#optimistic-pros-and-cons) can be vulnerable to. | Validity proofs are intense to compute – not worth it for applications with little on-chain activity. |
+| Secure and decentralized, since the data that is needed to recover the state is stored on the layer 1 chain.      |                                                                                                       |
 
 #### Use ZK rollups {#use-zk-rollups}
 
 - [Loopring](https://loopring.org/#/)
 - [Starkware](https://starkware.co/)
-- [Matter Labs zkSync](https://matter-labs.io/)
+- [Matter Labs zkSync](https://zksync.io/)
```
Comment (wackerow) [inline src/content/developers/docs/layer-2-scaling/index.md:82]: Is there a reason for this link switch when the former still works?

Comment (StanislavBreadless) [reply src/content/developers/docs/layer-2-scaling/index.md:82]: Matter Labs is the company.
zkSync is the zkRollup solution itself.

I believe that most users will be more interested in zkSync than the company behind it.

Comment (wackerow) [reply src/content/developers/docs/layer-2-scaling/index.md:82]: Ok cool, I'd have to agree.

**Review by wackerow — APPROVED** [review]
@StanislavBreadless LGTM... if anyone out there disagrees with any of the changes please feel free to comment, merging it in the meantime 👍

**Comment by wackerow** [comment]
@all-contributors please add @StanislavBreadless for content

**Pull request merged**